.github/workflows/integration-tests.yml

```diff
@@ -25,6 +25,7 @@ jobs:
           [
             github.com/ChainSafe/gossamer/dot/rpc/modules,
             github.com/ChainSafe/gossamer/lib/babe,
+            github.com/ChainSafe/gossamer/dot/sync,
```

Comment:
You might want to rebase your branch on development to get the missing required workflows for the `dot` and `dot/core` package integration tests

Reply:
Good point, rebased.